### ISSUE TITLE
[enhancement] include type in options

### ISSUE THREAD

**Issue by ageorgios** [RedKenrok/node-audiorecorder#3]
i would be great if you added the output type as an option

**Comment by RedKenrok** [comment]
Could you elaborate on what exactly you mean with this? For example what input you would like to provide and what you expected outcome should look like.

Kind regards,
Ron Dekker

**Comment by ageorgios** [comment]
something like:

```
const options = {
  // type of output.
  'output_type' = 'mp3'
};
```

**Comment by RedKenrok** [comment]
Ok, I see what you mean. I'll do some experimenting with opening up the `bits`, `encoding`, and `type` options.

**Comment by RedKenrok** [comment]
Version 1.1.5 is now available (58f9482d5aa7762f47ec4778c17e3d05b7269f17). You should now be able to change the `bits`, `encoding`, and `type` options. Which might give you the outcome you are looking for.

Kind regards,
Ron Dekker

**Comment by ageorgios** [comment]
thanks you are great!
